### ISSUE TITLE
How to upload multiple images in Cloudinary using nodeJS

### ISSUE THREAD

**Issue by KalaiselvanMahendran** [cloudinary/cloudinary_npm#108]
Here i have four images, in my router post i need to upload four images in cloudinary. And get call back to store a cloudinary url in database.

Am able to do this for single image, but there is no document to upload multiple images

```
router.post('/api/products', cpUpload, function(req, res){
     console.log(req.files.product_image_1[0].path);
     console.log(req.files.product_image_2[0].path);
     console.log(req.files.product_image_3[0].path);
     console.log(req.files.product_image_4[0].path);
});
```

Thanks in advance

**Comment by roeeba** [comment]
Hi @KalaiselvanMahendran , Uploading multiple resources in a single POST request is currently not supported. You will have to loop through your product list and upload one resource at a time. 
Also note that as [Cloudinary URL are dynamic](http://cloudinary.com/documentation/image_transformations#delivering_images_using_dynamic_urls), it is not advised to store the actual URL in your database, but rather the public ID, version and other parameters separately.

**Comment by roeeba** [comment]
Closing this issue due to the time elapsed. Please feel free to either re-open the issue, contact our support at http://support.cloudinary.com or create a new ticket if you have any additional issues.

**Comment by adeelibr** [comment]
When can be the facility of multiple images be supported by cloudinary?

**Comment by d-mendoza** [comment]
Hi @adeelibr,
To upload multiple images, it can be done in parallel (up to 15 connections concurrently). Uploading multiple resources in a single call is not currently supported.

**Comment by waletayo** [comment]
kindly follow this tutorial to upload multiple image to cloudianry 

https://tycodez.hashnode.dev/upload-multiple-images-to-cloudinary-node-mongodb-express-ck2qn9jgq006qiws1ck5ci9gi
